### ISSUE TITLE
Turning on `racon_enable_cuda` tricks `make install` to install to the wrong path

### ISSUE THREAD

**Issue by SHuang-Broad** [lbcb-sci/racon#17]
Hi,
I want to report an issue that should be trivial to fix.

The background is that I am trying to build a docker image of Racon with CUDA.
So I followed the instructions on Readme.

I cloned the repo to `/tmp/`, then the following will run fine

```
cmake \
        -DCMAKE_BUILD_TYPE=Release \
        -Dracon_build_tests=ON \
        -Dracon_build_wrapper=ON \
        .. && \
make && make install && \
racon -h
```

But if I turn on `-Dracon_enable_cuda=ON`, racon is not installed to the expected path via `make install`, i.e.

```
cmake \
        -DCMAKE_BUILD_TYPE=Release \
        -Dracon_enable_cuda=ON \
        -Dracon_build_tests=ON \
        -Dracon_build_wrapper=ON \
        .. && \
make && make install && \
racon -h
```
instead `make install` installs the binaries to 

```
[100%] Linking CXX executable bin/racon
[100%] Built target racon
Install the project...
-- Install configuration: "Release"
-- Installing: /tmp/racon/build/install/bin/racon
```

as opposed to the expected

```
[100%] Linking CXX executable bin/rampler
[100%] Built target rampler
[  4%] Built target edlib_static
[ 38%] Built target zlibstatic
[ 48%] Built target spoa
[ 53%] Built target thread_pool
[ 68%] Built target racon
[ 72%] Built target gtest
[ 76%] Built target gtest_main
[ 91%] Built target racon_test
[100%] Built target rampler
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/bin/racon
-- Installing: /usr/local/bin/rampler
```

I hope this is trivial to fix.

Thanks.
Steve

**Comment by rvaser** [comment]
Hello Steve,
I checked the cmake file and I am not sure why this happens. I am not available in the next week, but I will try to get back to you as soon as possible.

Best regards,
Robert

**Comment by SHuang-Broad** [comment]
Thanks Robert!

Below is my docker file, if that helps:

```dockerfile
FROM gpuci/clara-genomics-base:cuda10.1-ubuntu18.04-gcc7-py3.5

RUN apt-get -qqy install cmake git && \
    cd /tmp && \
    git clone --recursive https://github.com/lbcb-sci/racon.git racon && \
    cd racon && mkdir build && cd build && \
    cmake \
        -DCMAKE_BUILD_TYPE=Release \
        -Dracon_build_tests=ON \
        -Dracon_enable_cuda=ON \
        .. && \
    make && make install

RUN cd /tmp/racon/build/bin && ls -lh && \
    chmod +x racon_test && ./racon_test
```

**Comment by rvaser** [comment]
Hi Steve,
sorry for the delay in our communication. A temporary fix is to add `-DCMAKE_INSTALL_PREFIX=/usr/local` to the `cmake` command. ClaraGenomicsAnalysis submodule that racon uses is updating the root CMakeLists install prefix. A permanent solution to this issue will be pushed in the near future :)

Best regards,
Robert

**Comment by SHuang-Broad** [comment]
No worries Robert.
Thanks for the suggested fix!

I guess this issue can be left open, and gets closed when the patch is issued.

Best

**Comment by rvaser** [comment]
Fixed in version `v1.4.14`!
